config/routes.rb

```diff
@@ -2,8 +2,7 @@
   mount JasmineRails::Engine => "/specs" if defined?(JasmineRails)
   root "dashboards#show"
 
-  resource :dashboard, only: [:show]
-  get "/dashboards/closed", to: "dashboards#closed", as: "dashboard_closed"
+  get "/dashboards/show(/:id)", to: "dashboards#show", as: :dashboard
```

Comment:
Url's should be ```/dashboard/active``` and ```/dashboard/closed``` as each user can only ever view 1 type of dashboard so it makes more sense to have it as a singular resource

Fine to keep the root url for each user as an alias for ```/dashboard/active```